ios/app/brave_core_main.mm

```diff
@@ -178,11 +181,32 @@ - (void)onAppEnterForeground:(NSNotification*)notification {
 }
 
 - (void)onAppWillTerminate:(NSNotification*)notification {
+  NSLog(@"TERMINATING BRAVE CORE");
```

Comment:
Can we move this app termination change to a separate PR and also if we're going to keep a log here can we use the Chromium logger